api/client/system/info.go

```diff
@@ -214,6 +214,7 @@ func runInfo(dockerCli *client.DockerCli) error {
 	}
 
 	fmt.Fprintf(dockerCli.Out(), "Live Restore Enabled: %v\n", info.LiveRestoreEnabled)
+	fmt.Fprintf(dockerCli.Out(), "Default Isolation: %v\n", info.Isolation)
```

Comment:
should we;

- only print this on Windows?
- print this in the same section / similar section as / below "Runtime"?